src/main/java/seedu/address/logic/filters/CarFilter.java

```diff
@@ -0,0 +1,60 @@
+package seedu.address.logic.filters;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.model.customer.Car;
+
+/**
+ * Filters Cars based on user specified filters carBrand, carType.
+ */
+public abstract class CarFilter extends AbstractFilter {
+    /**
+     * Creates a filter for Cars
+     * @param filterString
+     */
+    public CarFilter(String filterString) {
+        super(filterString.trim());
+        requireNonNull(filterString);
+    }
+
+    /**
+     * Parses filter command for c/, cp/ prefixes.
+     * @param car
+     * @return
+     */
+    public static Car parseCar(String car) {
+        String trimmedCar = car.trim();
+        String[] carDetails = trimmedCar.split("\\+");
```

Comment:
Does "+Subaru" produce the array ["","Subaru"]

Reply:
ig so, we can add this as a testcase handling later on no worries